project-base/storefront/components/Pages/CategoryDetail/CategoryDetailProductsWrapper.tsx

```diff
@@ -46,76 +31,16 @@ export const CategoryDetailProductsWrapper: FC<CategoryDetailProps> = ({ categor
                 gtmProductListName={gtmProductListName}
                 products={categoryListedProducts}
                 fetching={fetching}
+                loadMoreFetching={loadMoreFetching}
                 category={category}
                 gtmMessageOrigin={GtmMessageOriginType.other}
             />
             <Pagination
                 paginationScrollTargetRef={paginationScrollTargetRef}
                 totalCount={category.products.totalCount}
+                isWithLoadMore
+                hasNext={categoryProductsData?.products.pageInfo.hasNextPage}
```

Comment:
Wouldn't be better prop name?

```suggestion
                isWithNextPage={categoryProductsData?.products.pageInfo.hasNextPage}
```

Reply:
Decided to rename it to `hasNextPage`